common/rtc_manager_rviz_plugin/src/rtc_manager_panel.cpp

```diff
@@ -0,0 +1,314 @@
+
+  client_rtc_commands_ = raw_node_->create_client<CooperateCommands>(
+    "/api/external/set/rtc_commands", rmw_qos_profile_services_default);
+
+  for (size_t i = 0; i < auto_modes_.size(); i++) {
+    auto & a = auto_modes_.at(i);
+    // auto mode
+    a->enable_auto_mode_cli = raw_node_->create_client<AutoMode>(
+      enable_auto_mode_namespace_ + "/" + a->module_name, rmw_qos_profile_services_default);
+  }
+}
+
+void RTCAutoMode::onChangeToAutoMode()
+{
+  AutoMode::Request::SharedPtr request = std::make_shared<AutoMode::Request>();
+  request->enable = true;
+  enable_auto_mode_cli->async_send_request(request);
+  auto_manual_mode_label->setText("AutoMode");
+  auto_manual_mode_label->setStyleSheet("background-color: #00FFFF;");
+}
+
+void RTCAutoMode::onChangeToManualMode()
+{
+  AutoMode::Request::SharedPtr request = std::make_shared<AutoMode::Request>();
+  request->enable = false;
+  enable_auto_mode_cli->async_send_request(request);
+  auto_manual_mode_label->setText("ManualMode");
+  auto_manual_mode_label->setStyleSheet("background-color: #FFFF00;");
+}
+
+void RTCManagerPanel::onRTCStatus(const CooperateStatusArray::ConstSharedPtr msg)
+{
+  rtc_table_->clearContents();
+  if (msg->statuses.empty()) return;
+  std::vector<CooperateStatus> coop_vec;
+  std::copy(msg->statuses.cbegin(), msg->statuses.cend(), std::back_inserter(coop_vec));
+  std::sort(
+    coop_vec.begin(), coop_vec.end(), [](const CooperateStatus & c1, const CooperateStatus & c2) {
+      return c1.start_distance < c2.start_distance;
+    });
+  // this is to stable rtc display not to occupy too much
+  size_t min_display_size{5};
+  size_t max_display_size{10};
+  rtc_table_->setRowCount(std::max(min_display_size, std::min(coop_vec.size(), max_display_size)));
+  int cnt = 0;
+  for (auto status : msg->statuses) {
+    // uuid
+    {
+      std::stringstream uuid;
+      uuid << std::setw(4) << std::setfill('0') << static_cast<int>(status.uuid.uuid.at(0));
+      auto label = new QLabel(QString::fromStdString(uuid.str()));
+      label->setAlignment(Qt::AlignCenter);
+      label->setText(QString::fromStdString(uuid.str()));
+      rtc_table_->setCellWidget(cnt, 0, label);
+    }
+
+    // module name
+    {
+      std::string module_name = getModuleName(status.module.type);
+      auto label = new QLabel(QString::fromStdString(module_name));
+      label->setAlignment(Qt::AlignCenter);
+      label->setText(QString::fromStdString(module_name));
+      rtc_table_->setCellWidget(cnt, 1, label);
+    }
+
+    // is aw safe
+    {
+      std::string is_safe = Bool2String(status.safe);
+      auto label = new QLabel(QString::fromStdString(is_safe));
+      label->setAlignment(Qt::AlignCenter);
+      label->setText(QString::fromStdString(is_safe));
+      rtc_table_->setCellWidget(cnt, 2, label);
+    }
+
+    // is operator safe
+    {
+      std::string is_safe = Bool2String(status.command_status.type);
```

Comment:
`status.command_status.type` is not `Bool`.

https://github.com/tier4/tier4_autoware_msgs/blob/tier4/universe/tier4_rtc_msgs/msg/Command.msg

Could you change the text from "True" or "False" to "Activate" or "Deactivate"?

Reply:
@rej55 
I think this should be the same as current expression for the same reason.
https://github.com/autowarefoundation/autoware.universe/pull/1936#discussion_r979517483

Reply:
I modufied to wait / execute 
[a00cf14](https://github.com/autowarefoundation/autoware.universe/pull/1936/commits/a00cf1456a9bc52b77f7a39c51f656470318592d)